frontend/src/components/pages/CourseOverview.tsx

```diff
@@ -48,14 +49,14 @@ const ViewCourse = (): React.ReactElement => {
             <Button onClick={onReturnToCourseOverviewClick} variant="link">
               <ChevronLeftIcon color="brand.royal" mr="15px" boxSize={30} />
               <Text variant="display-xs" color="brand.royal">
-                Return to Course browsing
+                Return to Course Browsing
               </Text>
             </Button>
           </Box>
-          <Text variant="display-lg" color="text.default" mb="16px !important">
+          <Text variant="display-lg" color="text.default" style={{ marginBottom: '16px' }}>
             {courseData?.course?.title}
           </Text>
-          <Text variant="body" color="text.default" mb="16px !important">
+          <Text variant="body" color="text.default" style={{ marginBottom: '16px' }}>
```

Comment:
Wondering, is there any reason a plain `mb=16px` doesn't work? If this is necessary then so be it, but for my own learning I'd like to understand why

Reply:
From inspect element, I was able to gather that when I use "mb", the styling is overwritten by a default style. However, using "style" makes this the changes dominant. Similarly, I am unsure why this particular behavior exists for the mb attribute.

Reply:
Ah, found the problem! `VStack` applies its own styling which is overwriting your margins. I actually think it's better to just set the spacing on the VStack instead of adding margins, though. `spacing={3}` looks good to my eyes.